internal/store/postgres/audit.go

```diff
@@ -0,0 +1,52 @@
+package postgres
+
+import (
+	"time"
+
+	"github.com/jmoiron/sqlx/types"
+	"github.com/raystack/shield/core/audit"
+)
+
+type Audit struct {
+	ID     string `db:"id"`
+	OrgID  string `db:"org_id"`
+	Source string `db:"source"`
+	Action string `db:"action"`
+
+	Actor    types.NullJSONText `db:"actor"`
+	Target   types.NullJSONText `db:"target"`
+	Metadata types.NullJSONText `db:"metadata"`
```

Comment:
Can we add a sequence column for each org as well? Suppose if someone wants to filter logs for a specific org, using this field they can get the logs in order

Reply:
Alternatively, we can filter with time (ascending/descending) in `auditlog_repository`, in current list logs function it will be returned in random order in my opinion

Reply:
They can use `start_time` and `end_time` for the range. But yeah let me add sorting based on `created_at`.